scripts-utils/epub3-utils/src/main/resources/xml/epub3-vocab.xsl

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0"
```

Comment:
it might be useful to add a "cleanup" function that tries to clean and fix a prefix declaration that contain validation errors into something valid?

Reply:
Hmm OK but what kind of errors would it detect?

Reply:
merging duplicate prefix, replacing redeclared reserved prefixes with default ones, removing prefixed default vocabs?

Reply:
OK, so you mean more like a function or template on the whole document rather than a function that parses a single prefix attribute? That could be useful yes. I'm definitely going to add more stuff to this library, notably there is some NLB code that I would like to integrate (or replace) and that does things similar to what you are proposing.

Reply:
You should now be able to do this kind of cleanup with the new [`px:epub3-pub-add-prefix`](https://github.com/daisy/pipeline-modules/commit/82adb09dd9bc8837be3b4cdfe8d65c34117b81d9) or [`px:epub3-pub-merge-prefix`](https://github.com/daisy/pipeline-modules/commit/f0b6caade7973071b94e5b0ca8f17af9cd04fde9) step.